src/main/java/filter/expression/QualifierType.java

```diff
@@ -19,15 +19,27 @@
 
     private static Map<String, QualifierType> initialiseAliases() {
```

Comment:
Does this Map contains all aliases? If not, at least add a comment to explain what it excludes and why.